backend/utils.py

```diff
@@ -79,6 +79,8 @@ def recursive_crud(model, values):
         item_id = vals.pop('id', None)
         if vals:
             stored = model.read([item_id], list(vals.keys()))[0]
+            if 'id' in stored.keys():
+                stored.pop('id', None)
```

Comment:
If you use `pop('id', None)` you don't need to check if the key exists. Simplt do `stored.pop('id', None)`